src/template.js

```diff
@@ -225,7 +242,7 @@ switch (data.method) {
 
     logger(data.method, viewedFiltersOptions);
     aa(data.method, viewedFiltersOptions);
-
+    data.gtmOnSuccess();
     break;
   }
 
```

Comment:
should there be a data.gtmOnFailure if no valid method is provided (for line 251)?